multisrc/overrides/mangathemesia/komiktap/src/Komiktap.kt

```diff
@@ -1,11 +1,49 @@
 package eu.kanade.tachiyomi.extension.id.komiktap
 
+import app.cash.quickjs.QuickJs
 import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
-import eu.kanade.tachiyomi.network.interceptor.rateLimit
-import okhttp3.OkHttpClient
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.Cookie
+import okhttp3.Interceptor
+import okhttp3.Response
+import java.io.IOException
 
 class Komiktap : MangaThemesia("Komiktap", "https://komiktap.me", "id") {
-    override val client: OkHttpClient = super.client.newBuilder()
-        .rateLimit(4)
-        .build()
+    override val client by lazy {
```

Comment:
Making the client lazy is unnecessary since it will be used as soon as the user enters the extension anyways. Also, why was the rate limit removed?